examples/aspnet/ADR.csproj

```diff
@@ -1,14 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <PropertyGroup>
-    <TargetFramework>net9.0</TargetFramework>
-    <Nullable>enable</Nullable>
-    <ImplicitUsings>enable</ImplicitUsings>
-  </PropertyGroup>
-
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
CLRF naar LF changes. Waarschijnlijk omdat jouw andere PR was aangemaakt voordat de `.gitattributes` in `develop` zaten